sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div>
+    <div class="row">
+      <header-title-value
+        title="Submitted date"
+        :value="
+          headerDetails.submittedDate
+            ? dateOnlyLongString(headerDetails.submittedDate)
+            : '-'
+        "
+      />
+      <div
+        v-if="headerDetails.institutionName"
+        class="mx-2 vertical-divider"
+      ></div>
+      <header-title-value title="Institution"
+        ><template #value
+          ><span class="link-primary" @click="navigateToInstitutionProfile()">
+            {{ headerDetails.institutionName }}
+          </span>
+        </template></header-title-value
+      >
+      <div
+        v-if="headerDetails.locationName"
+        class="mx-2 vertical-divider"
+      ></div>
+      <header-title-value
+        v-if="headerDetails.locationName"
+        title="Location"
+        :value="headerDetails.locationName"
+      />
+    </div>
+    <div class="row mt-1" v-if="showReviewDetails">
+      <div
+        class="row pl-4"
+        v-if="
+          headerDetails.status ===
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent
+        "
+      >
+        <div class="mr-2">
+          <v-icon icon="fa:fas fa-times-circle" size="20" color="error" />
+        </div>
+        <header-title-value
+          :title="reviewLabel.assessedByLabel"
+          value="Student"
+        />
+        <div class="mx-2 vertical-divider"></div>
+        <header-title-value
+          :title="reviewLabel.assessedDateLabel"
+          :value="dateOnlyLongString(headerDetails.updatedDate)"
+        />
+      </div>
+      <div
+        class="row pl-4"
+        v-if="
+          headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.Approved ||
+          headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.DeclinedBySABC
+        "
+      >
+        <div class="mr-2">
+          <v-icon
+            icon="fa:fas fa-times-circle"
+            v-if="
+              headerDetails.status ===
+              ApplicationOfferingChangeRequestStatus.DeclinedBySABC
+            "
+            size="20"
+            color="error"
+          />
+          <v-icon
+            icon="fa:fas fa-check-circle"
+            v-if="
+              headerDetails.status ===
+              ApplicationOfferingChangeRequestStatus.Approved
+            "
+            size="20"
+            color="success"
+          />
+        </div>
```

Comment:
Can't this section be simplified, maybe the logic can be done in the scripts and the template can receive the variable that needs to be populated, something like this, 
![image](https://github.com/bcgov/SIMS/assets/77353155/3528fd0d-a060-486c-99f3-c3c306b10eb6)